addons/account/tests/test_account_move_entry.py

```diff
@@ -491,7 +493,7 @@ def test_journal_sequence_ordering(self):
         self.assertEqual(copies[5].name, 'XMISC/2019/00004')
 
         # Can't have twice the same name
-        with self.assertRaises(ValidationError):
+        with self.assertRaises(psycopg2.IntegrityError), mute_logger('odoo.sql_db'), self.env.cr.savepoint():
```

Comment:
Wouldn't stacking make it more readable?
```suggestion
        with self.assertRaises(psycopg2.IntegrityError), \
                mute_logger('odoo.sql_db'), \
                self.env.cr.savepoint():
```
I usually don't like the `\`, but it helps in cases like this one, where adding parentheses is not an option.

Reply:
I prefer it like it is now.
I would definitely like it better with parenthesis in python3.9 though. https://www.python.org/dev/peps/pep-0617/#some-rules-are-not-actually-ll-1

Reply:
BTW, `mute_logger` can also be used as a decorator on the test method.
Decorating the method allows you to remove it from the `with` statement.

Reply:
I don't like muting more than I need to, the same way I don't like to put more than needed in a `try` clause, or catching too many exceptions in the `catch` clause.